mod/wet4/start.php

```diff
@@ -78,8 +78,8 @@ function wet4_theme_init()
 
 	//register a page handler for friends
 	elgg_unregister_page_handler('friends'); //unregister core page handler
-	elgg_unregister_page_handler('dashboard'); //unregister dashboard handler to make our own
-	elgg_register_page_handler('dashboard', 'wet4_dashboard_page_handler');
+	//elgg_unregister_page_handler('dashboard'); //unregister dashboard handler to make our own
```

Comment:
Wouldn't you want to keep the other one unregistered as well as not registering the wet4 one?

Reply:
Yes, we can